README.md

```diff
@@ -27,12 +27,9 @@ The project aims to:
   * [x] Administrator Role Compatible
   * [x] Multi-User Chat Generation (Multiple users chatting at the same time) - This was built in from Ollama `v0.2.1+`
   * [x] Automatic and Manual model pulling through the Discord client
-* [ ] Allow others to create their own models personalized for their own servers!
-  * [ ] Documentation on creating your own LLM
-  * [ ] Documentation on web scrapping and cleaning
```

Comment:
Are we totally scrapping this? We could probably create some documentation for this or link other work that does this.

Reply:
I was thinking so, I can drum up a quick doc on the Modelfile stuff for Ollama.

Reply:
I'm just going to link Ollama's docs on creating models, adding custom models, and the documentation for a `modelfile`.